### PR TITLE
CNI-11068; Add setup files;

### PR DESCRIPTION
**Pull request by jotshin**
JIRA ticket: https://conichi.atlassian.net/browse/CNI-11068

## Description
- Add setup files including Bundler, Cocoapods, and Git submodules.
-  The whole process of cloning, `make bootstrap`, and build of both targets are tested okay.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```
Comment (David-Henner) [inline Gemfile:1]: Should this commented line be kept  ?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+open:
+	open CNI-Itineraries.xcworkspace
+
+### Git commands
+
+pull:
+	git pull origin $(LOCAL_BRANCH)
+
+pr:
+	if [ "$(LOCAL_BRANCH)" == "develop" ]; then echo "${RED}In "$(LOCAL_BRANCH)", not PRing${RESET}"; elif [ "$(LOCAL_BRANCH)" == "master" ]; then echo "${RED}In "$(LOCAL_BRANCH)", not PRing ${RESET}"; else git push origin "$(LOCAL_BRANCH):$(BRANCH)"; open "https://github.com/conichiGMBH/CNI-Booking/compare/develop...$(BRANCH)"; fi
```
Comment (David-Henner) [inline Makefile:64]: `https://github.com/conichiGMBH/CNI-Booking/compare/develop`

Should it be `CNI-Booking` ?

**Review by David-Henner — COMMENTED** [review]
Thanks for the great work 👍

File: Gemfile (hunk 1 above)
Comment (jotshin) [reply Gemfile:1]: I didn't know it, but it appears that it's meant to be there:
https://stackoverflow.com/questions/37799296/what-does-the-comment-frozen-string-literal-true-do

File: Makefile (hunk 2 above)
Comment (jotshin) [reply Makefile:64]: Thanks. I'll ask @Ch3D's help on changing the repo's name first to see if it's viable to do the pod name change.

**Review by David-Henner — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (jotshin) [reply Makefile:64]: @David-Henner FYI since the Github repo renaming will affect the current users, we won't change it for now.

**Pull request merged**